src/components/FetchProductList.jsx

```diff
@@ -0,0 +1,140 @@
+import axios, { isAxiosError } from "axios";
+import { useEffect, useState } from "react";
+
+export default function FetchProductList() {
+  const [products, setProducts] = useState([]);
+  const [error, setError] = useState("");
+  const [isLoading, setIsLoading] = useState(true);
+
+  useEffect(() => {
+    const Promise = axios.get(
```

Comment:
you can use chaining instead of storing promise in a separate variable.
**Example**
`
axios.get().then().catch()
`